treescript/src/treescript/versionmanip.py

```diff
@@ -30,9 +30,10 @@
 }
 
 _VERSION_CLASS_PER_END_OF_SOURCE_REPO = {
+    "android-components": MavenVersion,
     "fenix": MobileVersion,
+    "firefox-android": MobileVersion,
```

Comment:
I noticed this and had it on my to-do list - glad you sorted it out.

Reply:
GMTA! 😃